pkg/csi-common/driver_test.go

```diff
@@ -31,93 +31,22 @@ const (
 )
 
 var (
-	fakeVersion           = csi.Version{Major: 5, Minor: 2, Patch: 0}
-	fakeVersionsSupported = []*csi.Version{
-		{
-			Major: 4, Minor: 0, Patch: 0,
-		},
-		{
-			Major: 4, Minor: 1, Patch: 0,
-		},
-	}
+	vendorVersion = "0.2.0"
 )
 
 func NewFakeDriver() *CSIDriver {
-	fakeVersion = csi.Version{Major: 5, Minor: 2, Patch: 0}
-	fakeVersionsSupported = []*csi.Version{
-		{
-			Major: 4, Minor: 0, Patch: 0,
-		},
-		{
-			Major: 4, Minor: 1, Patch: 0,
-		},
-	}
-
-	driver := NewCSIDriver(fakeDriverName, &fakeVersion, fakeVersionsSupported, fakeNodeID)
+
+	driver := NewCSIDriver(fakeDriverName, vendorVersion, fakeNodeID)
 
 	return driver
 }
 
 func TestNewFakeDriver(t *testing.T) {
 	// Test New fake driver with invalid arguments.
-	d := NewCSIDriver("", &fakeVersion, fakeVersionsSupported, fakeNodeID)
+	d := NewCSIDriver("", vendorVersion, fakeNodeID)
 	assert.Nil(t, d)
 }
 
-func TestCheckVersion(t *testing.T) {
```

Comment:
keep this test

Reply:
changes in CSI spec removed sending version in its calls, so everything was refactored to remove it.

Reply:
this test doesn't test CSI calls, it tests internal version string.